src/Chainweb/Version.hs

```diff
@@ -554,15 +563,18 @@ window FastTimedCPM{} = Nothing
 window Development = Just $ WindowWidth 120
 -- 120 blocks, should take 1 hour given a 30 second BlockRate.
 window Testnet02 = Just $ WindowWidth 120
-
-txSilenceDates :: ChainwebVersion -> Maybe (Time Micros)
-txSilenceDates Test{} = Nothing
-txSilenceDates TimedConsensus{} = Nothing
-txSilenceDates PowConsensus{} = Nothing
-txSilenceDates TimedCPM{} = Nothing
-txSilenceDates FastTimedCPM{} = Nothing
-txSilenceDates Development = Nothing
-txSilenceDates Testnet02 = Nothing
+window Mainnet01 = Just $ WindowWidth 120
+
+txSilenceEndDate :: ChainwebVersion -> Maybe (Time Micros)
+txSilenceEndDate Test{} = Nothing
+txSilenceEndDate TimedConsensus{} = Nothing
+txSilenceEndDate PowConsensus{} = Nothing
+txSilenceEndDate TimedCPM{} = Nothing
+txSilenceEndDate FastTimedCPM{} = Nothing
+txSilenceEndDate Development = Nothing
+txSilenceEndDate Testnet02 = Nothing
+-- Thursday, 2019 December 5, 12:00 AM
+txSilenceEndDate Mainnet01 = Just . Time $ TimeSpan 1575504000000000
```

Comment:
@slpopejoy @buckie @mightybyte This will disable mining coordination and new cut submission the instant this time is passed. If stopped, a node can't be restarted after this date either. Are we concerned about sudden drop-offs of Nodes/Miners?

Reply:
Is there a value to have many stalled nodes in the network? Those nodes would still validate and forward data, but what's the value for the network?

Less nodes mean a smaller diameter and lower latencies. More nodes mean more protection against partitions and attacks.

Assuming that we, and others, trust our bootstrap nodes, the benefits of a larger network don't seem that important.

Reply:
We have no choice but to shut them down, if they don't switch we'll just have to deal with it.